SpectraLogic.EscapePodClient/Utils/SafeExecutor.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2018 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using SpectraLogic.EscapePodClient.Exceptions;
+using SpectraLogic.EscapePodClient.Model;
+using System;
+using System.Net;
+
+namespace SpectraLogic.EscapePodClient.Utils
+{
+    internal class SafeExecutor
```

Comment:
What makes this executor safe?

Reply:
Maybe this is a bad name, its safe because it catches the action exception (shrug)
I can change it to EscapePodClientExceutor, sounds better?